package/gluon-config-mode-contact-info/i18n/fr.po

```diff
@@ -16,11 +16,16 @@ msgstr "Informations de Contact"
 msgid ""
 "Please provide your contact information here to allow others to contact "
 "you. Note that this information will be visible <em>publicly</em> on "
-"the internet together with your node's coordinates."
+"the internet together with your node's coordinates. This information is "
+"not required to operate a node. If you chose to enter data, it will be "
+"stored on this node and can be deleted by yourself at any time."
```

Comment:
It is relevant that this data is also collected in a central location (map server), and potentially cached for a very long time?

The FFHL meshviewer will show all information of offline nodes for several months after a node has disappeared. Not caching this information at all would solve the problem, but it would also make the information less useful, as you can't contact the operator of an unexpectedly offline node.

Reply:
DSGVO beschreibt "dauerhafte Speicherung". A cache is no permanently saved data (the closest translation I can come up with). OTOH several months defenitely is an issue wrt DSGVO because the intent of shutting down a node is not clear. It could have been done to not do Freifunk any more or to just stop providing their data. Ithink that however is a question of sane defaults for meshviewer  not for gluon.
At the same time it probably does not hurt to mention cache retention time (which should be in the order of days at most.)

Reply:
in #gluon, rotanid mentioned that this would not help because the data may be processed by anyone. Really, it is public, which the text already states.

In my rework I will attempt to make the consequence of this data being public more obvious.

Reply:
and done.